packages/core/src/utils.ts

```diff
@@ -40,3 +40,22 @@ export type AnyCase<T extends string> = Uppercase<T> | Lowercase<T>;
 export function isKey<T extends {} | []>(key: string | number | symbol, of: T): key is keyof T {
   return key in of;
 }
+
+/**
+ * Function to encode a number as hex in format low to high values
+ * @param {[Number]} input 
+ * @param {[Number]} length 
+ * @returns {[String]} hex value low-high order
+ */
+export function intLowHigh(input: number, length: number = 1): String {
```

Comment:
```ts
 if (input <= 0 || input % 1 !== 0) {
    throw new Error('Input must be a positive integer')
  }
```

Reply:
Added in 8d76c8306a0f31bc298dac945ca2bae4acea621d 

But in my opinion 0 would be valid input. If i need to have 00 00 for a command as in setMargin it would make sense to get the hex values 00 00 for the input 0 if the length for the hex couple is set for example to 2.